### ISSUE TITLE
import all objects from package for modules

### ISSUE THREAD

**Issue by mmuurr** [rticulate/import#25]
In the 'modules' use case, it'd be useful to import all objects from a package, effectively just like `library()`, but only have those exposed objects be visible to the current module.
For example, in `module.R`:
```R
library(tibble)
library(dplyr)
x <- function() {
  as_tibble(iris) %>%
    group_by(Sepal.Length) %>%
    summarize(mean_petal_width = mean(Petal.Width))
}
```
Then in some higher-level script, one would import the module:
```R
import::from("module.R", x)
x()  ## doesn't work because the attached packages have been detached
```
Rather than `library()`, as you've pointed out in the docs, we can use:
```R
import::here(as_tibble, .from = "tibble")
import::here(group_by, summarize, .from = "dplyr")
```
... but that can get cumbersome if the module uses LOTS of objects.
Is there the possibility of an `import::from(packagename, .all = TRUE)` option, that acts similarly to `library` but helps with this module use case?

**Comment by torfason** [comment]
It seems like this issue is very closely related to #23.

**Comment by torfason** [comment]
This has now been fixed on master. If you could install the most current version and test whether the fix works for you (and if the new version works for you in general), that would be greatly appreciated. You can get the fix with:
`remotes::install_github("smbache/import")`

The new `.all` and `.except` parameters should work with both packages and modules:

```
# Import all functions from the module_base.R package (part of unit tests for import)
import::from(module_base.R, .all=TRUE)

# Using .except implies .all=TRUE by default
import::from(module_base.R, .except=c("fun2","fun3")) 
``` 

[ fixed-in-master : Issue to be closed following CRAN release ]

**Comment by torfason** [comment]
Fixed in release 1.2.0
